CHANGELOG.md

```diff
@@ -10,9 +10,11 @@ SPDX-License-Identifier: LGPL-3.0-or-later
 - Updated license information ([#138](https://github.com/ikarus-project/ikarus/pull/138))
 - Added detailed documentation for ikarus-examples ([#140](https://github.com/ikarus-project/ikarus/pull/140))
 - Added computation of Cauchy stress in linear elasticity ([#137](https://github.com/ikarus-project/ikarus/pull/137))
-- Added greeting and init function for reasonable default, e.g. log also to file ([#147](https://github.com/ikarus-project/ikarus/pull/147))
+- Added greeting and init function for a reasonable default, e.g., log also to file ([#147](https://github.com/ikarus-project/ikarus/pull/147))
 - Added an interface for the material library ([#154](https://github.com/ikarus-project/ikarus/pull/154))
 - Added a wrapper for flat and blocked basis ([#157](https://github.com/ikarus-project/ikarus/pull/157))
+- Fixed the `greenLagrangeStrains` expression and refactored tests for faster compilation in `dune-localfefunctions` ([#6](https://github.com/ikarus-project/dune-localfefunctions/pull/6))
```

Comment:
The changelog here should only iclude Ikarus information and not deun-localfefunctions